core/polyfills.php

```diff
@@ -281,6 +281,7 @@ function mb_strtolower($str)
     'mpg' => 'video/mpeg',
     'mpeg' => 'video/mpeg',
     'mov' => 'video/quicktime',
+    'flv' => 'video/x-flv',
```

Comment:
flv is already defined as video/x-flv on line 261

Reply:
Weird, it was missing in my copy. Must have been a merge mixup somewhere.